src/main/java/seedu/address/logic/parser/EditCommandParser.java

```diff
@@ -68,28 +63,16 @@ public EditCommand parse(String args) throws ParseException {
         if (argMultimap.getValue(PREFIX_FAMILY).isPresent()) {
             editPersonDescriptor.setFamily(ParserUtil.parseFamily(argMultimap.getValue(PREFIX_FAMILY).get()));
         }
-        parseTagsForEdit(argMultimap.getAllValues(PREFIX_TAG)).ifPresent(editPersonDescriptor::setTags);
-
+        if (argMultimap.getValue(PREFIX_TAG).isPresent()) {
+            editPersonDescriptor.setTags(ParserUtil.parseTags(argMultimap.getAllValues(PREFIX_TAG)));
+        }
+        if (argMultimap.getValue(PREFIX_REMARK).isPresent()) {
+            editPersonDescriptor.setRemark(ParserUtil.parseRemark(argMultimap.getValue(PREFIX_REMARK).get()));
```

Comment:
I realised that if you had ever put a remark for the person, currently there is no functionality to clear the remark under edit?

meaning to say that once you put a remark for a client there must always be a remark for that person. 
not sure if this is something we might want to consider in future

Reply:
Nvm it works well